src/twisted/conch/test/test_ckeygen.py

```diff
@@ -141,7 +141,7 @@ def test_printFingerprint(self):
             'format': 'md5-hex'})
         self.assertEqual(
             self.stdout.getvalue(),
-            '768 3d:13:5f:cb:c9:79:8a:93:06:27:65:bc:3d:0b:8f:af temp\n')
+            '2048 85:25:04:32:58:55:96:9f:57:ee:fb:a8:1a:ea:69:da temp\n')
```

Comment:
were these tests failing?

I would say to just add a new 2048 key size and use that one in the failing tests, and let the remaining tests as they are